docs/guides/integration/docker.md

```diff
@@ -340,18 +340,31 @@ WORKDIR /app
 RUN --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,source=uv.lock,target=uv.lock \
     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
-    uv sync --frozen --no-install-project
+    uv sync --frozen --no-install-project --link-mode=copy --compile-bytecode
 
 # Copy the project into the image
 ADD . /app
 
 # Sync the project
 RUN --mount=type=cache,target=/root/.cache/uv \
-    uv sync --frozen
+    --mount=type=bind,source=uv.lock,target=uv.lock \
+    --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
```

Comment:
Wouldn't these be added by the `ADD . /app` invocation?

Reply:
Oh my, I blame tunnel vision. 🤦 

In my case I added only the `src` directory because `ADD . /app` adds a lot of things that I don't want to add. Thinking about it, with `--link-mode=copy`, I could probably even `--mount` the source directory itself as well?

Reply:
The source directory needs to stick around because we don't support non-editable installs of workspace members during syncs yet (#5792). Maybe it'd be best to address your case in the documentation by adding a case to the optimization examples for copying the src directory instead of the full application folder?